src/globals/api/apiCall.ts

```diff
@@ -5,7 +5,7 @@ import { LogError } from '@/globals/api/LogError'
  * @param object - Любой объект
  */
 function convertObjectToURLSearchParams(object: object) {
-  const filtered = Object.entries(object).filter(([_, value]) => value !== undefined)
+  const filtered = Object.entries(object).filter(([_, value]) => value !== undefined && value !== null)
   return new URLSearchParams(filtered)
```

Comment:
Преврати в многострочный код, а то стало тяжело читать:
```suggestion
  const filtered = Object.entries(object).filter(([_, value]) => {
    return value !== undefined && value !== null
  })
```